### ISSUE TITLE
selectAtoms() requires loaded coordinates even for pure "topology" information

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#270]
`AtomGroup.selectAtoms()` requires that coordinates are loaded even though many selections should just work with a topology.

This is not a big deal but it would be nice if the following just worked (because I expect that I can select topology data in the absence of coordinates):

```python
import MDAnalysis; from MDAnalysis.tests.datafiles import PSF
u = MDAnalysis.Universe(PSF)
p = u.selectAtoms("protein")
```

but that currently raises `AttributeError`: *No trajectory loaded into Universe*

**Comment by richardjgowers** [comment]
I imagine this is because some selections won't work (eg around) without coordinates. I guess what needs doing is some selections being tagged as "need coords" , and that error message only being thrown if you hit one of them.

**Comment by mnmelo** [comment]
I think it's slightly less simple because the base Selection class already accesses universe.coord, irrespective of the selection being used.
Shouldn't be too hard to fix, though. I'm on it.

**Comment by richardjgowers** [comment]
This is fixed in the issue-363 branch.

**Comment by jbarnoud** [comment]
@richardjgowers 
Thanks! I was complaining about it just yesterday.
